app/src/main/kotlin/com/hippo/ehviewer/ui/screen/SearchBarScreen.kt

```diff
@@ -241,6 +244,15 @@ fun SearchBarScreen(
     } else {
         Modifier
     }
+
+    // Workaround IME not hiding after performing search
+    // https://github.com/FooIbar/EhViewer/pull/412
+    val view = LocalView.current
+    val keyboardManager = remember { SoftwareKeyboardControllerCompat(view) }
+    SideEffect {
+        if (!active) keyboardManager.hide()
+    }
+
```

Comment:
It's not working on API 25.
Search bar is active at the beginning since it's focused.